MaxText/layers/linears.py

```diff
@@ -49,6 +50,7 @@
 RMSNorm = normalizations.RMSNorm
 Quant = quantizations.AqtQuantization
 
+MESH_FSDP_AXIS = "fsdp"
```

Comment:
I believe this should correspond to all batch dimensions, not just fsdp, e.g. should be the list [dp, fsdp, fsdp_transpose] give [here](https://github.com/google/maxtext/blob/f50f5ed2673d0c5bdf10ff06ed6a71d39a629080/MaxText/configs/base.yml#L125)

You should use use nn_logical_to_mesh((BATCH, None, None)) as done for flash attention
https://github.com/google/maxtext/blob/f50f5ed2673d0c5bdf10ff06ed6a71d39a629080/MaxText/layers/attentions.py#L198

Can you also leave a comment here that Model parallelism and sequence parallelism are not currently supported with this megablox kernel? Additionally we should assert this somehwere (I think in pyconfig) - if a user requests megablox with either TP or sequence parallelism we should error out saying this is currently not supported

Reply:
Sg. Let me have a try and add assertion.

Reply:
Updated. Thanks!